packages/backend/src/clientKeys/model.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseModel } from '../shared/baseModel'
+
+import { JWKWithRequired } from 'auth'
+
+export class ClientKeys extends BaseModel {
+  public static get tableName(): string {
+    return 'clientKeys'
+  }
+
+  // The id should be the same as the kid.
```

Comment:
Is this still true as of the discussion from this morning? i.e. isn't the `id` the internal database id and the `kid` a fully-qualified URL that contains the `id`?

Reply:
Yes, will update the comment.